### ISSUE TITLE
New absent terms in PATO

### ISSUE THREAD

**Issue by nicolevasilevsky** [pato-ontology/pato#331]
On the POTATO call on 07/30/30, we decided we should:

- Create new terms:
   - physical absence (child of morphology)
   - absent process (child of process quality)

In the short term:
- Keep PATO_0000462 absent for now
    - Add a comment- "We recommend using the new absent terms physical absence and absent process, respectively. This term is a legacy term  and will ultimately be obsoleted."
- Announce we will be deprecating ‘absent’ and will switch to using the new terms

- Create new PATO classes: (@nicolevasilevsky)
- [ ] morphology -> physical absence 
- [ ] process quality -> absent process

- [ ] Fix patterns (@chris-grove)
- [ ] Fix imports in respective ontologies

- [ ] obsolete PATO_0000462 absent, after the new terms have been adopted by the community.

cc @matentzn 
cc @ybradford

**Comment by nicolevasilevsky** [comment]
@chris-grove commented in slack:
I seem to recall that after our journal club discussion with @davidos on Sept 10, 2020 (see minutes) that the latest proposal superseded PATO ticket #331, but maybe I'm misremembering

https://docs.google.com/document/d/1WrQanAMuccS-oaoAIb9yWQAd4Rvy3R3mU01v9wHbriM/edit#bookmark=id.2p6d3tw070x

and @dosumis  commented:
I have some half finished modelling experiments I've been meaning to present on a uPheno call.  Can we put this on the agenda for the meeting on the 25th.

We'll discuss this on the 25th.

**Comment by nicolevasilevsky** [comment]
@pnrobinson said in slack:
I do not understand the difference between physical absence (applied to something in the body) and anatomical entity absence

**Comment by ybradford** [comment]
I agree with @chris-grove comment that we discussed this on 9/10/2020 and moved toward using 'has fewer parts of type'. I believe tickets #341 and #340 are stubs meant to deal with the suggested changes.

**Comment by ddooley** [comment]
Where is a modeling example of this?  How do I say (an instance) patient X has an absence of a right kidney?  

And while we are on the subject, how do I express that an instance of a kidney is severed from Patient X (and not connected to any body).
